### PR TITLE
docs: March 2020 blog

### PR DESCRIPTION
**Pull request by daveconde**

### CODE REVIEW

File: content/en/blog/news/cjxd-202003.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Another area that some of the team are working in right now is the implementation of the depreciation policy, PRs are being lined up to remove commands that have been market for [deprecation](https://jenkins-x.io/commands/deprecation/). This will help cut down the code surface area a bit and make Jenkins X a bit easier to maintain. 
+
+
+The [labs team](https://github.com/jenkins-x-labs) have also been extremely busy experimenting with [helm3](https://helm.sh/blog/helm-3-released/) and [helmfile](https://github.com/roboll/helmfile) with a view to using it as a foundation for Multi Cluster support in Jenkins X.
+We still have a bit to learn before we start to roll this in to Jenkins X. One of the next areas we'll be experimenting with is what the upgrade process from a helm2 environment to a helm3 environment will look like. 
+You can jump over to the new [labs documentation](https://jenkins-x.io/docs/labs/) for more information on experiments and you can find the labs issue tracker [here](https://github.com/jenkins-x-labs/issues/issues)
```
Comment (markyjackson-taulia) [inline content/en/blog/news/cjxd-202003.md]: `labs` should be `lab's`

File: content/en/blog/news/cjxd-202003.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+If you dont use terraform and you have another way of provisioning infrastructure don't worry, we'll be providing lots of information in the docs on exactly what you will need to create before booting your Jenkins X.
+
+Another area that some of the team are working in right now is the implementation of the depreciation policy, PRs are being lined up to remove commands that have been market for [deprecation](https://jenkins-x.io/commands/deprecation/). This will help cut down the code surface area a bit and make Jenkins X a bit easier to maintain. 
+
+
+The [labs team](https://github.com/jenkins-x-labs) have also been extremely busy experimenting with [helm3](https://helm.sh/blog/helm-3-released/) and [helmfile](https://github.com/roboll/helmfile) with a view to using it as a foundation for Multi Cluster support in Jenkins X.
+We still have a bit to learn before we start to roll this in to Jenkins X. One of the next areas we'll be experimenting with is what the upgrade process from a helm2 environment to a helm3 environment will look like.
```
Comment (markyjackson-taulia) [inline content/en/blog/news/cjxd-202003.md]: `in to` should be `into`

File: content/en/blog/news/cjxd-202003.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+The idea behind this enhancement is that some companies may require the infrastructure to be provisioned by a user (someone in operations perhaps) that requires a different set of permissions than the user that runs `jx boot` to install Jenkins X.
+Previously this happened when a user ran `jx boot`. But from community feedback we learned that some users would like this process split. 
+To help with this we are in the process of developing terraform modules that you can use to create the necessary resources. 
+Work on this has been moving really quickly and we have been focusing on support for GKE, EKS & AKS.
+We are really excited to see the awesome contributions from [Heba Elayoty](https://github.com/helayoty) of [Microsoft](https://www.microsoft.com) on adding support for [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/) in Jenkins X.
+If you dont use terraform and you have another way of provisioning infrastructure don't worry, we'll be providing lots of information in the docs on exactly what you will need to create before booting your Jenkins X.
```
Comment (markyjackson-taulia) [inline content/en/blog/news/cjxd-202003.md]: `dont` should be `don't`

File: content/en/blog/news/cjxd-202003.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+<img src="/images/logo/cloudbees.png"/>
+</figure>
+
+March has been a really busy month for us working on Jenkins X, several exciting projects are now well underway. 
+
+Recently at office hours we discussed a [proposal to improve how we provision](https://github.com/jenkins-x/enhancements/tree/master/proposals/3) the cloud resources (storage buckets, service accounts & cryptographic keys) that Jenkins X requires.
+The idea behind this enhancement is that some companies may require the infrastructure to be provisioned by a user (someone in operations perhaps) that requires a different set of permissions than the user that runs `jx boot` to install Jenkins X.
+Previously this happened when a user ran `jx boot`. But from community feedback we learned that some users would like this process split. 
+To help with this we are in the process of developing terraform modules that you can use to create the necessary resources. 
+Work on this has been moving really quickly and we have been focusing on support for GKE, EKS & AKS.
```
Comment (rawlingsj) [inline content/en/blog/news/cjxd-202003.md]: would be good to link to the repos and share how the OSS community can get involved

Comment (rawlingsj) [reply content/en/blog/news/cjxd-202003.md]: any links to docs / blogs / demos etc might help too

**Pull request merged**